apps/content/services/admin/asset_recitation_ayah_timestamps_upload_service.py

```
@@ -163,9 +164,12 @@ def bulk_upload_recitation_ayah_timestamps(asset_id: int, files: Iterable) -> Re
                 updated_total += len(to_update)
 
     except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)